plugin/adsk/scripts/AEmayaUsdProxyShapeTemplate.mel

```diff
@@ -1,7 +1,66 @@
+global proc AEmayaUsdProxyShapeInfoNew(string $input)
+{
+    textFieldGrp -label "Root Layer" -editable 0
+                 -ann "Identifies the root layer of a stage. If a file path is shown in this field, the root layer is a file on disk. If a layerName is shown in this field, the root layer is an anonymous layer."
+                 mayaUsdProxyShapeRootLayer;
+
+    textFieldGrp -label "Default Prim" -editable 0
+                 -ann "As part of its metadata, each stage can identify a default prim. This is the primitive that is referenced in if you reference in a file."
+                 mayaUsdProxyShapeDefaultPrim;
+
+    AEmayaUsdProxyShapeInfoReplace($input);
+}
+
+global proc AEmayaUsdProxyShapeInfoReplace(string $input)
+{
+    // From the input attribute we simply want the node name.
+    string $nodeName = plugNode($input);
+    string $res[] = `ls -l $nodeName`;
+    string $fullNodeName = $res[0];
+
+    // Call python helpers to get the info we want to display.
+    string $rootLayer   = `python("import AETemplateHelpers; AETemplateHelpers.GetRootLayerName('" + $fullNodeName + "')")`;
+    string $defaultPrim = `python("import AETemplateHelpers; AETemplateHelpers.GetDefaultPrimName('" + $fullNodeName + "')")`;
+
+    textFieldGrp -edit -text $rootLayer mayaUsdProxyShapeRootLayer;
+    textFieldGrp -edit -text $defaultPrim mayaUsdProxyShapeDefaultPrim;
+}
+
 global proc AEmayaUsdProxyShapeTemplate( string $nodeName )
 {
     editorTemplate -beginScrollLayout;
 
+        //
+        // Proxy Shape specific attributes
+        //
+        editorTemplate -suppress "complexity";
+        editorTemplate -suppress "inStageData";
+        editorTemplate -suppress "stageCacheId";
+        editorTemplate -suppress "outStageCacheId";
+
+        editorTemplate -beginLayout "Stage" -collapse 0;
+            // We aren't displaying the "inStageData" attribute in the ProxyShapeInfo. We simply
+            // pass it in order to get the current nodename. Cannot use the input one here as it
+            // changes based on which proxy shape the AE is viewing.
+            editorTemplate -callCustom "AEmayaUsdProxyShapeInfoNew" "AEmayaUsdProxyShapeInfoReplace" inStageData;
```

Comment:
This adds the "Root Layer" and "Default Prim" (non-editable) fields. These aren't actual attributes on the proxy shape. The fields are for display purposes only.